patches/react-native-draggable-flatlist+4.0.1.patch

```diff
@@ -0,0 +1,94 @@
+diff --git a/node_modules/react-native-draggable-flatlist/src/components/DraggableFlatList.tsx b/node_modules/react-native-draggable-flatlist/src/components/DraggableFlatList.tsx
```

Comment:
This patch file comes from [here](https://github.com/computerjazz/react-native-draggable-flatlist/issues/539#issuecomment-2299111102). Someone already created [a PR](https://github.com/computerjazz/react-native-draggable-flatlist/pull/551) containing these changes to be added into `react-native-draggable-flatlist` but for now we need to handle this error using a patch.